### PR TITLE
feat: @wagmi/vue

### PR DESCRIPTION
**Pull request by tmm**
### Description

Adds `feat: @wagmi/vue` package

### CODE REVIEW

File: packages/vue/src/composables/useBlockNumber.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export function useBlockNumber<
 
   const queryOptions = computed(() => {
     const {
-      chainId = configChainId.value,
+      chainId = configChainId,
```
Comment (tmm) [inline packages/vue/src/composables/useBlockNumber.ts:78]: not reactive unless `.value` is removed, but messes with the type for `get*QueryOptions`

Comment (tmm) [reply packages/vue/src/composables/useBlockNumber.ts:78]: need to figure out a nice way to use existing `get*QueryOptions` while also unwrapping these type of framework-specific values (in this case `ref`)

Related https://github.com/TanStack/query/pull/7390

Comment (tmm) [reply packages/vue/src/composables/useBlockNumber.ts]: resolved https://github.com/wevm/wagmi/pull/3906/commits/ff73c95510ae256febf693f58a5bdb30214964f8

**Comment by reslear** [comment]
wow amazing

**Comment by tmm** [comment]
@reslear agree!

**Pull request merged**